src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -9,21 +9,47 @@ entries:
 
 {{<label storage_mining_subsystem>}}
 
-TODO:
+# Filecoin Storage Mining Subsystem
 
-- rename "Storage Mining Worker" ?
+The Filecoin Storage Mining Subsystem ensures a storage miner can effectively commit storage to the Filecoin protocol in order to both:
 
-Filecoin Storage Mining Subsystem
+- participate in the Filecoin {{<sref storage_market>}} by taking on client data and participating in storage deals.
+- participate in Filecoin {{<sref storage_power_consensus>}}, verifying and generating blocks to grow the Filecoin blockchain and earning block rewards and fees for doing so.
+
+The above involves a number of steps to putting on and maintaining online storage, such as:
+
+- Committing new storage (see Sealing and PoRep)
+- Continously proving storage ({see {<sref election_post>}})
+- Declaring storage faults and recovering from them.
+
+## Storage States
+
+When managing their storage sectors as part of Filecoin mining, storage providers will account for where in the mining cycle their sectors are. For instance, has a sector been committed? Does it need a new PoSt?
+
+Through this cycle, sectors will fall into either a miner's:
+- *PreCommitted Sectors*: New sectors to be committed using PoRep
+- `Proving Set`: Committed sectors to be proven over time using PoSts
+  - Sectors can be faulted by the miner itself (`Declared Faults`).
+- `Fault Set`: Committed sectors that have failed to be proven.
+  - Fault recovery allows sectors to return to the `Proving Set`.
```

Comment:
Blocking: to be precise, `RecoverFaults`, which can be called any time by the miner, will move sectors from `Failing` to `Recovering` which is part of the `ProvingSet`.

Reply:
why do we actually need the `Recovering` state? Just an optimization? (We could always test every failing sector each time)...

Reply:
Recovering state will be transitioned into Active State after a PoSt, it's a required state.

Reply:
why not move `Failing` into `Active` automatically in that instance is what I mean?